src/main/java/io/micronaut/build/testresources/StopTestResourcesHelper.java

```diff
@@ -75,4 +91,18 @@ public FileVisitResult postVisitDirectory(Path dir, IOException exc) throws IOEx
         });
     }
 
+    private Path getServerSettingsDirectory() {
+        if (shared) {
+            return ServerUtils.getDefaultSharedSettingsPath();
+        }
+        return serverSettingsDirectoryOf(buildDirectory.toPath());
+    }
+
+    private Path getKeepAliveFile() {
+        return getServerSettingsDirectory().resolve("keepalive");
+    }
+
+    private Path serverSettingsDirectoryOf(Path buildDir) {
+        return buildDir.resolve("../.micronaut/test-resources");
```

Comment:
Would putting it here mean that it will need to be added to .gitignore by the user. I know `target` tends to be ignored by default?

Reply:
Good point. Would you want me to do that via starter?

Reply:
If so, I guess we should add it to the one generated for maven projects in starter?

Reply:
✨

Reply:
Done in https://github.com/micronaut-projects/micronaut-starter/pull/1297/commits/c9b4273248dbfdda18d49ae7f46729c5f38c7381 😎

Reply:
That's a good catch, @timyates !

Reply:
Sometimes, I surprise myself 🤣